docs/js/copybutton.js

```diff
@@ -0,0 +1,147 @@
+// introduces special behavior for ShellSession
```

Comment:
What is the origin and license for thise code?

Reply:
The original code has MIT license and is included with the pip install of the sphinx-copybutton. In order to change the behavior, I changed a couple of lines and overwrite the original with this.

Reply:
For future reference: https://github.com/executablebooks/sphinx-copybutton

> In order to change the behavior, I changed a couple of lines and overwrite the original with this.

Can you commit the original, unmodified version separately from your modifications? If we ever get into the situation where we need to rebase onto a newer upstream version, then it would be good to know what the local modifications were. In you local changes you can then add a link to the original file and it's license.

Reply:
Is it enough to include a link to the repo and license along with comments calling out my changes in the modified version? I can set the version number in the requirements.txt to avoid problems with upstream changes.